apache-airflow-mixin/panels.libsonnet

```diff
@@ -0,0 +1,134 @@
+local g = (import './g.libsonnet');
+local commonlib = import 'common-lib/common/main.libsonnet';
+
+{
+  new(this)::
+    {
+      local signals = this.signals,
+
+      //
+      // Overview Dashboard Panels
+      //
+      dagFileParsingErrorsPanel:
+        commonlib.panels.generic.timeSeries.base.new('DAG file parsing errors', targets=[signals.overview.dagParsingErrors.asTarget()])
+        + g.panel.timeSeries.panelOptions.withDescription('The number of errors from trying to parse DAG files in an Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      slaIssuesPanel:
+        commonlib.panels.generic.timeSeries.base.new('SLA misses', targets=[
+          signals.overview.taskSlaMisses.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The number of Service Level Agreement misses for any DAG runs in the Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      taskFailuresPanel:
+        commonlib.panels.generic.timeSeries.base.new('Task failures', targets=[
+          signals.overview.taskFailures.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The overall task instances failures for an Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('none'),
+
+      dagSuccessDurationPanel:
+        commonlib.panels.generic.timeSeries.base.new('DAG success duration', targets=[
+          signals.overview.dagAvgSuccessDuration.asTarget() { interval: '2m' },
+        ])
+        + g.panel.timeSeries.panelOptions.withDescription('The average time taken for recent successful DAG runs by DAG ID in the Apache Airflow system.')
+        + g.panel.timeSeries.standardOptions.withUnit('s')
+        + g.panel.timeSeries.options.legend.withPlacement('right')
```

Comment:
Should this also use a table legend, i.e.: `+ g.panel.timeSeries.options.legend.withDisplayMode('table')`?